wunderio/next4drupal-project
# Nex 6 storybook

*Link to ticket: [NEX-6](https://wunder.atlassian.net/jira/software/c/projects/NEX/boards/389?modal=detail&selectedIssue=NEX-6)*

*Changes proposed in this PR:*
- Storybook is added to the project
- Articles stories are added as an example

*How to test:*
1. cd /next
2. run npm run storybook
3. check Articles stories
